src/authhandler.cpp

```diff
@@ -51,61 +51,36 @@ const QString AUTH_PATH             ("AuthPath");
 const QString TOKEN_PATH            ("TokenPath");
 const QString REDIRECT_URI          ("RedirectUri");
 const QString HOST                  ("Host");
-const QString SLASH                 ("/");
-const QString AUTH                  ("auth");
-const QString AUTH_METHOD           ("method");
-const QString MECHANISM             ("mechanism");
 
-AuthHandler::AuthHandler(Accounts::Manager *manager, const quint32 accountId, const QString &accountService, QObject *parent)
+AuthHandler::AuthHandler(Accounts::AccountService *service, QObject *parent)
```

Comment:
It's generally a little suspicious when this keeps a pointer to something it doesn't own. 

At minimum this could maybe have some comment that he service is not owned by the instance, but could maybe consider having it stored in shared pointers.

Reply:
You're right. I put the AccountService from CalDavClient under a QSharedPointer().